helm/templates/user-configmap.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongodb-user-configmap
+data:
+    mongo-users.js: |
+      print("Started Adding the Users.");
+      db = db.getSiblingDB("admin");
+
+      // works for version < 5.x.x (based on mongo version 4.4.18)
+      const mongo_root_password = _getEnv('MONGO_INITDB_ROOT_PASSWORD');
+      const mongo_service_password = _getEnv('MONGO_INITDB_SERVICE_PASSWORD');
+      const mongo_developer_password = _getEnv('MONGO_INITDB_DEVELOPER_PASSWORD');
+      
+      db.auth('{{ .Values.mongodbRootUsername }}', mongo_root_password)
+
+
+      // Check if the service user already exists
```

Comment:
Instead of duplicating these, can you create a function "createUserIfNotExists()" and passing-in the username?
Will be helpful in the future for introducing more users.